policyengine_uk/parameters/gov/dwp/targeted_childcare_entitlement/child_age_eligible.yaml

```diff
@@ -0,0 +1,25 @@
+description: The Department for Education provides targeted childcare entitlement in England for specific age brackets
+metadata:
+  type: single_amount
+  threshold_unit: year
+  reference:
+    - title: The Local Authority (Duty to Secure Early Years Provision Free of Charge) Regulations 2014 - regulation 3 
+      href: https://www.legislation.gov.uk/uksi/2014/2147/regulation/3/made
+    - title: Document for 15 hours free education and childcare for 2-year old
```

Comment:
`title` should always be as close as possible to just the official title, like you have above. GOV.UK | Claim benefits for free childcare etc